includes/Data/Themes.php

```diff
@@ -79,7 +85,7 @@ private static function check_approved( $value ) {
 	  * @return array
 	  */
 	public static function step_preview_data() {
-		$theme_step_data   = Patterns::get_count_of_patterns();
+		$theme_step_data = Patterns::get_count_of_patterns();
```

Comment:
Hey Sorry, but on Line 74 is self::$wp_slugs present in this file?

Reply:
Nopeee, great catch! can't believe this lived on for so long, all the more reason to do code reviews earlier.